app/_src/policies/introduction.md

```diff
@@ -1,6 +1,650 @@
 ---
 title: Policies
 ---
+{% if_version gte:2.9.x %}
+## What is a policy?
+
+A policy is a set of configuration that will be used to generate the proxy configuration.
```

Comment:
this sentence uses twice word `proxy` and once `dataplane`, should it be changed to :
- `proxy` → `data-plane proxy`
- `dataplane configuration` → `Dataplane resource` 
to be compliant with the concepts page?